oaknational/Oak-Web-Application
# PUPIL-204: enable order questions on the quiz

## Description

Music year: [2022](https://www.youtube.com/watch?v=rg18Kf4en2o)

Enables answering of order questions in the new pupil quiz journey. This takes the order question [UI developed in oak-components](https://github.com/oaknational/oak-components/pull/122) and applies it to the pupil quiz.

Figma: https://www.figma.com/file/G7z5ufo2T0CHk14CkadeuB/Pupils-high-fidelity-designs?type=design&node-id=6942-85574&mode=dev

## How to test

1. Go to https://deploy-preview-2291--oak-web-application.netlify.thenational.academy/pupils/programmes/english-secondary-ks4-eduqas/units/writing-masters/lessons/writing-excellent-descriptions
2. Take the starter and/or exit quizzes
3. You should see an order question in each
4. Try re-ordering the items and verify that the grading and feedback works as expected.

## Screenshots

<img width="1693" alt="Screenshot 2024-03-05 at 17 34 31" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/3e51f6e6-790d-4523-a42b-817ecaaf9872">
<img width="1693" alt="Screenshot 2024-03-05 at 17 55 20" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/4acf9e71-ac42-45ab-a54c-e76155eb82ce">
<img width="1693" alt="Screenshot 2024-03-05 at 17 54 40" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/5b2aa5d6-efd6-47bf-bec6-24151055bce4">
<img width="1693" alt="Screenshot 2024-03-05 at 17 54 18" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/012e71c1-ca18-45c1-a09d-88ea814f7830">
